src/coreComponents/constitutive/solid/Damage.hpp

```diff
@@ -182,11 +288,20 @@ class DamageUpdates : public UPDATE_BASE
     #endif
   }
 
-  arrayView2d< real64 > const m_damage;
+  GEOSX_HOST_DEVICE
+  virtual real64 getBiotCoefficient( localIndex const k ) const
+  {
+    return m_biotCoefficient[k];
+  }
+
+  arrayView2d< real64 > const m_newDamage;
+  arrayView3d< real64 > const m_damageGrad;
   arrayView2d< real64 > const m_strainEnergyDensity;
+  arrayView2d< real64 > const m_volStrain;
   real64 const m_lengthScale;
   real64 const m_criticalFractureEnergy;
   real64 const m_criticalStrainEnergy;
+  arrayView1d< real64 > const m_biotCoefficient;
```

Comment:
I don't think the `biotCoefficient` should be stored in the Damage model. Are you using a `PorousSolid`? I would assume that you are if there is a poroelastic solver.

Reply:
I think it is a bit strange to have the biotCoefficient here as well. However, I don't need to use a PorousSolid for this problem, so, if I were to use one, I would need to turn off the flow solvers. I believe it lead to more modifications in the code, so, I did this approach instead.